resources/ports.yml

```
@@ -1080,6 +1080,11 @@ ports:
         name: Yakuake
         category: terminal
         platform: [linux]
+    yazi:
+        name: Yazi
+        category: cli
+        platform: agnostic
```

IIRC, `agnostic` was meant to capture all platforms including `ios` and `android` which I don't think makes sense for yazi. 

I know we have our libraries as agnostic but that's more because it's harder to define what platforms something like `python` or `java` doesn't run on lol.

```suggestion
        platform: [linux, macos, windows]
```

---

Maybe add a `android` here? - Yazi can be run on Android, https://github.com/termux/termux-packages/blob/master/packages/yazi/build.sh

---

Will do that